src/test/fixtures/composes.tsx

```diff
@@ -386,45 +574,64 @@ export const mockClones = (composeId) => {
           },
         },
       ],
+      meta: {
+        count: 4,
+      },
+      links: {
+        first: '',
+        last: '',
+      },
+    };
+  } else {
+    return {
+      data: [],
+      meta: {
+        count: 0,
+      },
+      links: {
+        first: '',
+        last: '',
+      },
     };
   }
-
-  return null;
 };
 
-export const mockCloneStatus = {
-  'f9133ec4-7a9e-4fd9-9a9f-9636b82b0a5d': {
-    options: {
-      ami: 'ami-0e778053cd490ad21',
-      region: 'us-west-1',
-    },
-    status: 'success',
-    type: 'aws',
-  },
-  '48fce414-0cc0-4a16-8645-e3f0edec3212': {
-    options: {
-      ami: 'ami-9f0asd1tlk2142124',
-      region: 'us-west-1',
-    },
-    status: 'success',
-    type: 'aws',
-  },
-  '0169538e-515c-477e-b934-f12783939313': {
-    options: {
-      ami: 'ami-9fdskj12fdsak1211',
-      region: 'us-west-2',
-    },
-    status: 'failure',
-    type: 'aws',
-  },
-  '4a851db1-919f-43ca-a7ef-dd209877a77e': {
-    options: {
-      ami: 'ami-9fdskj12fdsak1211',
-      region: 'eu-central-1',
-    },
-    status: 'success',
-    type: 'aws',
-  },
+export const mockCloneStatus = (cloneId: string): UploadStatus => {
+  const mockClones = {
+    'f9133ec4-7a9e-4fd9-9a9f-9636b82b0a5d': {
+      options: {
+        ami: 'ami-0e778053cd490ad21',
+        region: 'us-west-1',
+      },
+      status: 'success',
+      type: 'aws',
+    } as UploadStatus,
+    '48fce414-0cc0-4a16-8645-e3f0edec3212': {
+      options: {
+        ami: 'ami-9f0asd1tlk2142124',
+        region: 'us-west-1',
+      },
+      status: 'success',
+      type: 'aws',
+    } as UploadStatus,
+    '0169538e-515c-477e-b934-f12783939313': {
+      options: {
+        ami: 'ami-9fdskj12fdsak1211',
+        region: 'us-west-2',
+      },
+      status: 'failure',
+      type: 'aws',
+    } as UploadStatus,
+    '4a851db1-919f-43ca-a7ef-dd209877a77e': {
+      options: {
+        ami: 'ami-9fdskj12fdsak1211',
+        region: 'eu-central-1',
+      },
+      status: 'success',
+      type: 'aws',
+    } as UploadStatus,
+  };
+  return mockClones[cloneId as keyof typeof mockClones];
```

Comment:
Typing the object will allow TS to infer this info.
```suggestion
return mockClones[cloneId];
```